.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Could we remove the Prettier config from this PR? Seems out of scope for the changes here.

Reply:
removed.